packages/blade/src/components/Typography/Code/__tests__/Code.native.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Code } from '../Code';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+
+describe('<Code />', () => {
+  it('should render Code with default properties', () => {
+    const { getByText, toJSON } = renderWithTheme(<Code>TEST_TOKEN</Code>);
+    expect(getByText('TEST_TOKEN')).toBeTruthy();
```

Comment:
Minor (optional): just `getByText` assertion is enough. I realized this later we don't need `toBeTruthy` because the test will fail if `getByText` fails.

Reply:
Ohh righttt!! removed the assertion